### PR TITLE
LPS-143388 Create PATCH UserGroup for User Group Headless API

### PR DESCRIPTION
**Pull request by erickmonteiroo**

### CODE REVIEW

**Comment by drewbrokke** [comment]
@erickmonteiroo this is probably a real test failure.

**Comment by erickmonteiroo** [comment]
ci:test:relevant

**Comment by erickmonteiroo** [comment]
ci:test:relevant

File: modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/UserGroupResourceImpl.java [hunk 1]
```diff
@@ -67,6 +67,19 @@ public UserGroup putUserGroup(Long userGroupId, UserGroup userGroup)
...
+		UserGroup userGroup, UserGroup existingUserGroup) {
+
+		if (userGroup.getDescription() != null) {
+			existingUserGroup.setDescription(userGroup.getDescription());
+		}
+
+		if (userGroup.getName() != null) {
+			existingUserGroup.setName(userGroup.getName());
+		}
+	}
```
Comment (drewbrokke) [inline modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/UserGroupResourceImpl.java:81]: Take a look at the `patchUserGroup` method in the `BaseUserGroupResourceImpl` class - it's basically taking care of this for you already :)

**Review by drewbrokke — CHANGES_REQUESTED** [review]
Just one comment @erickmonteiroo.

**Comment by erickmonteiroo** [comment]
ci:test:relevant

**Review by drewbrokke — APPROVED** [review]

**Comment by drewbrokke** [comment]
ci:forward

**Pull request closed without merging**